mmv1/third_party/terraform/website/docs/r/compute_security_policy.html.markdown

```diff
@@ -320,12 +320,24 @@ The following arguments are supported:
 
 * `layer_7_ddos_defense_config` - (Optional) Configuration for [Google Cloud Armor Adaptive Protection Layer 7 DDoS Defense](https://cloud.google.com/armor/docs/adaptive-protection-overview?hl=en). Structure is [documented below](#nested_layer_7_ddos_defense_config).
 
+* `auto_deploy_config` - (Optional) Configuration for [Automatically deploy Adaptive Protection suggested rules](https://cloud.google.com/armor/docs/adaptive-protection-auto-deploy?hl=en). 
```

Comment:
We often like to add a sentence like `Structure is [documented below](#auto_deploy_config).`

Reply:
Thank you once again, we already put this information on the documentation